js/plugins/i18next.js

```diff
@@ -8,8 +8,9 @@ export function install(Vue) {
         return i18n._(value, options);
     });
 
-    Vue.filter('dt', function(value, format) {
-        return value ? moment(value).format(format || 'LLL') : '-';
+    Vue.filter('dt', function(value, format, empty) {
+        empty = empty !== undefined ? empty : '-';
```

Comment:
Because I wanted to be backward compatible. If I change the signature, I need to modify every calls in the admin.
Datpicker are the only places where we need empty value to trigger placeholder display.